ND Supporting Document 2_2.adoc

```diff
@@ -1073,25 +992,6 @@ If the time period selection in FIA_AFL.1.2 is included in the ST, then the eval
 
 === TOE Access (FTA)
```

Comment:
maybe the app note needs to be updated to say if there is no local session, then this requirement is met by default